packages/video_player/video_player_avfoundation/example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -25,6 +25,6 @@
     <string>arm64</string>
   </array>
   <key>MinimumOSVersion</key>
-  <string>9.0</string>
+  <string>11.0</string>
```

Comment:
Please revert the auto-migration to the build files; those should be done separately, in conjunction with an minimum Flutter version update and some related changes.

Reply:
Could I have avoided that with some of the `flutter_plugin_tools` or is it just some extra care I need to take manually?

Anyway, I will revert it.

Reply:
It's not related to the repository tooling, `flutter` does this auto-migration because the current version no longer supports older versions of iOS. Updating the plugins for that change is still in progress.